flake8_quotes/__init__.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
```

Comment:
So maybe make 2 CLI options (`--quotes` and `--inline-quotes`) with `--quotes` as deprecated. This is prob getting too big for this PR. Maybe it should be a precursor PR?